cmd/minishift/cmd/addon/addon_install.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	unspecifiedSourceError   = "You need to specify the source for the add-on."
+	unspecifiedSourceError   = "You must specify the source directory path of the add-on."
```

Comment:
This is an error message. I bet you it will be forgotten then. There is hardly any loss in clarity here by being less specific. Also, installing from URL is high on the trio list. I would expect this to land in the next sprint or so. What's the point of "toggling" the message then?